### PR TITLE
Nuget deployment via Azure DevOps

### LINKED ISSUE #3: Create AppVeyor build and publish to IntelliTect MyGet feed.

### PR DESCRIPTION
**Pull request by adamskt**
Fixes #3 and #6.  Rather than coughing up a package every time someone commits to master, it will only push to Nuget when someone creates a tag off the release branch.  We will have to manually increment the version number in the YAML file until I can find a better way.

### CODE REVIEW

**Comment by adamskt** [comment]
We should squash & merge, unless you think folks like to watch my epic struggles with YAML.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ It provides real, authenticated Windows Auth capabilities - not just a mock of s
 
 ## Getting Started
 
-There is currently no NuGet package for this project. Open an issue if this project is of interest to you and we'll look at publishing a NuGet package. Otherwise, you can add this project as a git submodule to try it out.
+[![Build Status](https://intellitect.visualstudio.com/IntelliTect/_apis/build/status/IntelliTect.AspNetCore.TestHost.WindowsAuth-Nuget-Publish?branchName=master)](https://intellitect.visualstudio.com/IntelliTect/_build/latest?definitionId=50?branchName=master)
+
+Visit [Nuget.org](https://www.nuget.org/packages/IntelliTect.AspNetCore.TestHost.WindowsAut) to pull this library into your project as a dependency.
```
Comment (ascott18) [inline README.md]: Typo "WindowsAut" -> "WindowsAuth" in the Nuget.org URL.

File: README.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ It provides real, authenticated Windows Auth capabilities - not just a mock of s
 
 ## Getting Started
 
-There is currently no NuGet package for this project. Open an issue if this project is of interest to you and we'll look at publishing a NuGet package. Otherwise, you can add this project as a git submodule to try it out.
+[![Build Status](https://intellitect.visualstudio.com/IntelliTect/_apis/build/status/IntelliTect.AspNetCore.TestHost.WindowsAuth-Nuget-Publish?branchName=master)](https://intellitect.visualstudio.com/IntelliTect/_build/latest?definitionId=50?branchName=master)
```
Comment (ascott18) [inline README.md]: https://intellitect.visualstudio.com/IntelliTect/_build/latest?definitionId=50?branchName=master is requiring a sign-in - looks like https://intellitect.visualstudio.com/IntelliTect is private, so this link isn't viewable by anyone who isn't an IntelliTect person. Interesting, the build badge is still viewable without a sign-in.

We may want to make a new project on https://intellitect.visualstudio.com for builds of our miscellaneous GitHub projects - something like "IntelliTectPublic" perhaps.

**Review by ascott18 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (adamskt) [reply README.md]: Are public projects a generally-available feature in DevOps?

Comment (ascott18) [reply README.md]: Yes. Coalesce is, for example: https://intellitect.visualstudio.com/coalesce (open in incognito).

**Review by ascott18 — APPROVED** [review]

**Pull request merged**